src/components/SimpleChart.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  ColorType,
+  createChart,
+  type ISeriesApi,
+  type SingleValueData,
+  type Time,
+} from 'lightweight-charts'
+import { useEffect, useRef } from 'react'
+import { selectSelectedCurrency } from 'state/slices/selectors'
+import { store } from 'state/store'
+
+type SimpleChartProps<T extends number | Time> = {
+  data: SingleValueData<T>[]
+  seriesType: 'histogram' | 'line'
+}
+
+const backgroundColor = 'rgba(188, 214, 240, 0.04)'
+const lineColor = '#2962FF'
+const textColor = 'white'
+const areaTopColor = 'rgba(41, 98, 255, 0.5)'
+const areaBottomColor = 'rgba(41, 98, 255, 0.28)'
+
+const currentLocale = window.navigator.languages[0]
+const selectedCurrency = selectSelectedCurrency(store.getState())
+const priceFormatter = Intl.NumberFormat(currentLocale, {
+  style: 'currency',
+  currency: selectedCurrency,
+  notation: 'compact',
+  compactDisplay: 'short',
+}).format
+
+const chartContainerStyles = { width: '100%', height: '500px' }
+
+export const SimpleChart = <T extends number | Time>({ data, seriesType }: SimpleChartProps<T>) => {
+  const chartContainerRef = useRef<HTMLDivElement | null>(null)
+
+  useEffect(() => {
+    if (chartContainerRef.current && data) {
+      const chart = createChart(chartContainerRef.current, {
+        grid: {
+          vertLines: {
+            color: 'rgba(42, 46, 57, 0.5)',
+          },
+          horzLines: {
+            color: 'rgba(42, 46, 57, 0.5)',
+          },
+        },
+        layout: {
+          background: { type: ColorType.Solid, color: backgroundColor },
+          textColor,
+        },
+        width: chartContainerRef.current.offsetWidth,
+        height: chartContainerRef.current.offsetHeight,
+        localization: { priceFormatter },
+        timeScale: {
+          timeVisible: true,
+        },
+      })
+      const newSeries =
+        seriesType === 'line'
```

Comment:
nitpick - feels like line would be more of a default choice, or exhaustive switch for posterity?

Reply:
- ef8c69fb44f8998d8f7de02aa8347168094383a2 `Line` as default 
- 4437dec80e53bc78789b81fd848e18537344ec3b conforming to `SeriesType`
- 8cfcd3aab4 exhaustive series handling, and use the correct `Area` vernacular instead of line for area type
  - Area
  
<img width="1078" alt="image" src="https://github.com/shapeshift/web/assets/17035424/22bcfc41-f000-4ac6-8a79-f2ac3f09d6e6">

  - Histogram
  
<img width="1058" alt="image" src="https://github.com/shapeshift/web/assets/17035424/a7d6e2d0-79f1-4a3b-9c4d-a7c5cc59ac20">

  - Line
  
<img width="1070" alt="image" src="https://github.com/shapeshift/web/assets/17035424/e784a033-df42-4e09-aabe-e4bc81cd16e9">

  - Baseline
<img width="1064" alt="image" src="https://github.com/shapeshift/web/assets/17035424/b2c133d0-294f-4d44-99fb-73d5d82f4e6b">

  - Candlestick
<img width="1059" alt="image" src="https://github.com/shapeshift/web/assets/17035424/0fde8131-b077-46fd-917a-d2ff37e3eff9">
  - Bar
    
<img width="1066" alt="image" src="https://github.com/shapeshift/web/assets/17035424/50ba656d-575b-472f-a078-3302ae4155d2">

Reply:
:fire: